### ISSUE TITLE
View contents of a torrent when selected

### ISSUE THREAD

**Issue by valdotdev** [Tympanix/Electorrent#25]
Add the ability to view the contents of a torrent in the list especially if it contains multiple files. Usually a pane at the bottom would appear listing those.

And I'm not sure if this needs a separate issue but I guess this should also come with the ability to set bandwidth priority per file in the torrent.

**Comment by leptoid** [comment]
"Your program sucks. It doesn't..." Ok, I stole that from the FAQ.  
In addition to valdotdev's comment, it would be extremely helpful if you could not only see the individual files, but tell it whether to download or skip them as well.  Case in point, I have a 1TB Torrent I am downloading (mostly for seeding purposes) but I don't have the space for that on my seedbox, so I have to tell it not to get all of the files.  You program is super fast, works amazing, but is missing this feature so I have to go back to my seedbox's clunky/slow URL interface.
Thanks! 👍

**Comment by Tympanix** [comment]
I have some time on my hands so i could possibly implement this feature. With all clients supporting this feature it's going to a pretty substantial implementation tough. But you're right. It would be cool 👍

**Comment by valdotdev** [comment]
@Tympanix go at a pace that suits/works with you best! 👍 
Keep up the great work! 😄

**Comment by su30** [comment]
Hi! Do still plan to add this feature?

**Comment by tympanix** [comment]
Yes @su30 it is planned but I do not have time/resources to implement it at the moment considering we are now up at supporting 6 clients which would mean 6 times the work. I would like to see this one day as well though.

**Comment by JPUnD** [comment]
I'm just here to show my support for this feature and possible updates, as when this feature goes live I'll probably switch to this client.

**Comment by cerroverb** [comment]
With all due respect and appreciation for this software and its developers, I'm kind of flabbergasted that this hasn't been implemented yet. This sort of kills any use case I might have for it, and I otherwise really liked this client. Please consider adding it, even though it's 10 years later lo.

**Comment by tympanix** [comment]
Partially addressed in v2.10.0 by #511: the new details panel displays torrent files, progress, and sizes. Cross-client file selection and priority editing are not implemented, so this remains open.

**Comment by tympanix** [comment]
Closing this.

File selection and file priority will be scoped for the new issue
